### ISSUE TITLE
Anchor tags have incorrect _blank property

### ISSUE THREAD

**Issue by briankereszturi** [justin1954/justins-website#2]
Anchor tags have an attribute named "target" whose value is "_blank". Tags should be updated to reflect the correct form.

See http://www.w3schools.com/tags/tag_a.asp for a reference of the tag.

**Comment by justin1954** [comment]
didn’t you say that _blank opens a new tab

**Comment by briankereszturi** [comment]
Yes, but that's not how you use it. Check out the documentation and google it to see how others do it. It's a good idea to test your links to make sure they work before you commit them.

**Comment by justin1954** [comment]
ok

**Comment by justin1954** [comment]
is it better now?

**Comment by briankereszturi** [comment]
Much better! You can close this issue.

**Comment by justin1954** [comment]
hello ,can you send me a back round photo for my website?
I am looking for a picture like the one you used on that one webpage you showed me
